velox/docs/functions/presto/url.rst

```diff
@@ -14,6 +14,36 @@ The URL extraction functions extract components from HTTP URLs (or any valid URI
 
 The extracted components do not contain URI syntax separators such as ``:`` , ``?`` and ``#``.
 
+Consider for example the below URI:
+
+.. code-block::
+
+    http://www.ics.uci.edu/pub/ietf/uri/?k1=v1#Related
+
+    scheme    = http
+    authority = www.ics.uci.edu
+    path      = /pub/ietf/uri/
+    query     = k1=v1
+    fragment  = Related
+
+
+Invalid URI's
+-------------
+Well formed URI's should not contain ascii whitespace and percent-encoded URI's should be followed by two hexadecimal
```

Comment:
Add empty line.

ascii -> ASCII

Consider splitting this sentence into two for readability.

```
Well formed URI's should not contain ascii whitespace. Percent-encoded URI's should be followed by two hexadecimal ...

Is "Percent-encoded URI's" a well-known term? Consider adding a link to a definition in Wikipedia or some other good source.

Reply:
Will do. "Percent-encoded" is how its referred to in the URI RFC, I will reference that.